### PR TITLE
Fix null subtraction error

### PR DESCRIPTION
**Pull request by michielderoos**
**Describe the Pull Request**

We were trying to subtract from null. No longer!  

**Todo**

- [x] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [x] Request review from relevant @person or team
- [x] QA your pull request. This includes running the app, login, testing changes, checking [accessibility](https://www.a11yproject.com/checklist/) etc.
- [x] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver

### CODE REVIEW

File: app/migrations/versions/456eba67de05_.py [hunk 1]
```diff
@@ -18,9 +18,17 @@
 depends_on = None
 
 
+def index_exists(name):
```
Comment (michielderoos) [inline app/migrations/versions/456eba67de05_.py:21]: This change needs to be present for a deploy to work. It's already deployed in Pacific, but is still waiting for a review here: https://github.com/teamsempo/SempoBlockchain/pull/584

File: app/server/utils/transfer_limits/limits.py [hunk 2]
```diff
@@ -316,10 +316,11 @@ class AggregateTransferAmountMixin(object):
     @staticmethod
     def used_aggregator(transfer: CreditTransfer, query_constructor: QueryConstructorFunc):
         # We need to sub the transfer amount from the allowance because it's hard to exclude it from the aggregation
-        return query_constructor(
+        allowance = query_constructor(
             transfer,
             db.session.query(func.sum(CreditTransfer.transfer_amount).label('total'))
-        ).execution_options(show_all=True).first().total - int(transfer.transfer_amount)
+        ).execution_options(show_all=True).first().total or 0
+        return allowance - int(transfer.transfer_amount)
```
Comment (tristanhcole) [inline app/server/utils/transfer_limits/limits.py:323]: makes sense

**Review by tristanhcole — APPROVED** [review]
Tested locally and works for me 👍

**Comment by michielderoos** [comment]
[Fix null subtraction error](https://trello.com/c/cJLUO4LM/722-fix-null-subtraction-error)

**Pull request merged**